app/helpers/housekeeping.py

```diff
@@ -24,18 +24,19 @@ def run(self):
                 settings.reload_configuration_files()  # reload configuration file, in case new whitelisted items were added by the analyst, they should be processed!
 
                 try:
+                    logging.logger.info("housekeeping - going to remove all whitelisted outliers")
                     total_docs_whitelisted = es.remove_all_whitelisted_outliers()
 
                     if total_docs_whitelisted > 0:
                         logging.logger.info("housekeeping - total whitelisted documents cleared from outliers: " + str(total_docs_whitelisted))
                     else:
                         logging.logger.info("housekeeping - whitelist did not remove any outliers")
 
-                except Exception as e:
+                except Exception:
```

Comment:
Can you send this exception to sentry too?
```python
from sentry_sdk import capture_exception

try:
    a_potentially_failing_function()
except Exception as e:
    # Alternatively the argument can be omitted
    capture_exception(e)
```

Reply:
Tracked as issue, thanks